x-pack/docs/en/security/authorization/document-level-security.asciidoc

```diff
@@ -7,12 +7,12 @@ In particular, it restricts which documents can be accessed from document-based
 read APIs. 
 
 To enable document level security, you use a query to specify the documents that 
-each role can access. The document query is associated with a particular index 
+each role can access. The document query is associated with a particular data stream, index, 
 or index pattern and operates in conjunction with the privileges specified for 
-the indices.
+the data streams and indices.
```

Comment:
Minor: I suppose it's now "data stream, index, and index _or data stream_ pattern."